manifests/init.pp

```diff
@@ -27,14 +27,15 @@
   $conf_dir       = '/etc/mesos',
   # e.g. zk://localhost:2181/mesos
   $zookeeper      = '',
+  $quorum         = 1,
   # if "zk" is empty, master value is used
   $master         = '127.0.0.1',
   $master_port    = 5050,
   $owner          = 'root',
   $group          = 'root',
   # could be a fact like $::ipaddress or explicit ip address
-  $listen_address = undef,
-  $repo           = undef,
+  $listen_address = '0.0.0.0',
+  $repo           = 'mesosphere',
```

Comment:
Right now Mesosphere is best distribution channel for binary Mesos packages, I guess we can leave it as default.